src/app/validation/validation-experiments/validation-experiments.component.html

```diff
@@ -44,6 +44,19 @@ <h3 style="margin: 25px 0 20px 0;">1. Upload template</h3>
         Upload a File
       </button>
   </div>
+
+  <div fxLayout="row">
+    <mat-radio-group [(ngModel)]="action" fxLayout="row" [style.margin-top]="'20px'">
+      <mat-radio-button value="submission" [matTooltip]="tooltipSubmission"
+                        [matTooltipClass]="{'yellow-tooltip': true}"
+                        [matTooltipShowDelay]=1000
```

Comment:
The tooltip takes too long to appear with 1000ms delay. A slightly lesser delay of 500ms may be ideal as discussed.